input/pagecontent/vte1.md

```diff
@@ -13,22 +13,22 @@ The example presented is based on the QDM version of CMS108v7, which has been re
 
 |Resource Type|Profile Name|Link to Profile|
 |---|---|---|
-|Condition|QI Core Condition Problems Health Concerns Profile|[QI Core Condition Problems Health Concerns]|
-|Coverage|QI Core Coverage Profile|[QI Core Coverage]|
-|DeviceRequest|QI Core DeviceRequest Profile|[QI Core DeviceRequest]|
-|DeviceUseStatement|QI Core DeviceUseStatement Profile|[QI Core DeviceUseStatement]|
-|Encounter|QI Core Encounter Profile|[QI Core Encounter]|
+|Condition|QI-Core Condition Problems Health Concerns Profile|[QI-Core Condition Problems Health Concerns]|
```

Comment:
For specific profile names, use QICore instead of QI-Core to match their profile names.